crates/solidity/inputs/language/src/definition.rs

```diff
@@ -3794,7 +3794,7 @@ codegen_language_macros::compile!(Language(
                             definitions = [
                                 // Allows unicode characters:
                                 TokenDefinition(
-                                    enabled = Range(from = "0.4.12", till = "0.7.0"),
+                                    enabled = Till("0.7.0"),
                                     scanner = Sequence([
```

Comment:
I see the two tab "\t" tests are also affected. I wonder if it is included in these changes?

Reply:
Yes, the tabs where also valid starting from 0.4.11. Try compiling this:


```
pragma solidity ^0.4.11;
contract A {
	string abc = "tab:	";
}
```